src/styles.css

```
@@ -0,0 +1,55 @@
+  font-weight: 500;
+  cursor: pointer;
+  
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+
+  filter: var(--css-label-brightness);
+  pointer-events: none;
+  background-color: var(--css-label-background-color);
+  font-weight: 700;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

font-weight is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)